src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodec.java

```diff
@@ -363,6 +365,14 @@ static ByteBuffer byteBufferFromEJSON(DataType targetCQLType, EJSONWrapper wrapp
       }
       return ByteBuffer.wrap(binaryPayload);
     }
+
+    static InetAddress inetAddressFromString(String value) throws IllegalArgumentException {
+      try {
+        return InetAddress.getByName(value);
```

Comment:
Oh, so it handles invalid values for us.

Reply:
yes, ones that don't convert. Exception thrown here will be re-caught and give proper API error response.